online-fishing-store/src/app/main-layout/main-layout.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="body">
+  <app-header class="body_header"></app-header>
+  <div class="body_main">
```

Comment:
It is not a body. Why all sections is body???
It is also header, footer, content